common/src/main/java/com/wynntils/models/items/encoding/impl/block/CustomIdentificationDataTransformer.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.encoding.impl.block;
+
+import com.wynntils.core.WynntilsMod;
+import com.wynntils.core.components.Models;
+import com.wynntils.models.items.encoding.data.CustomIdentificationsData;
+import com.wynntils.models.items.encoding.type.DataTransformer;
+import com.wynntils.models.items.encoding.type.ItemTransformingVersion;
+import com.wynntils.models.stats.type.StatPossibleValues;
+import com.wynntils.models.stats.type.StatType;
+import com.wynntils.utils.UnsignedByteUtils;
+import com.wynntils.utils.type.ArrayReader;
+import com.wynntils.utils.type.ErrorOr;
+import com.wynntils.utils.type.RangedValue;
+import com.wynntils.utils.type.UnsignedByte;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+public class CustomIdentificationDataTransformer extends DataTransformer<CustomIdentificationsData> {
+    @Override
+    protected ErrorOr<UnsignedByte[]> encodeData(ItemTransformingVersion version, CustomIdentificationsData data) {
+        return switch (version) {
+            case VERSION_1 -> encodeCustomIdentificationData(data);
+        };
+    }
+
+    @Override
+    public ErrorOr<CustomIdentificationsData> decodeData(
+            ItemTransformingVersion version, ArrayReader<UnsignedByte> byteReader) {
+        return switch (version) {
+            case VERSION_1 -> decodeCustomIdentificationData(byteReader);
+        };
+    }
+
+    @Override
+    public byte getId() {
+        return 12;
```

Comment:
... and, to complement my previous comment, I'd like to see an ID field introduced here as well. (The "No magic numbers in code" rule).